fhir-server/liberty-config/server.xml

```diff
@@ -199,4 +199,7 @@
         </application-bnd>
     </webApplication>
     <include optional="true" location="${server.config.dir}/batchDs.xml"/>
+    
+    <!-- The key used for encryption of the internal JavaBatch Job instance/execution id which is used in bulkdata APIs  -->
+    <jndiEntry jndiName="config/bulkdatajobencryptionkey" value="change-password"/>
```

Comment:
I think this should go into the properties file separate from JNDI

Reply:
I didn't put it into any of the tenant config files because I want the same key to be used for all the tenants. I considered either server.xml or server.env, and I thought server.xml seems a better choice for me.

Reply:
changed to use fhir server config.